.buildkite/pipeline.yml

```diff
@@ -24,6 +24,19 @@ steps:
       - make build_swift
       - make build_ios_static
 
+  - label: Static Carthage build
+    timeout_in_minutes: 10
+    agents:
+      queue: opensource-mac-cocoa
+    concurrency: 3
+    concurrency_group: cocoa-unit-tests
+    env:
+      LANG: "en_GB.UTF-8"
+      DEVELOPER_DIR: "/Applications/Xcode.app/Contents/Developer"
```

Comment:
It would be clearer if the Xcode path included the version, i.e. `/Applications/Xcode_11.7.app`

FYI this is the naming convention [used by GitHub Actions.](https://github.com/actions/virtual-environments/blob/macOS-10.15/20201212.1/images/macos/macos-10.15-Readme.md#xcode)